### DISCUSSION TITLE
system won't load

### DISCUSSION THREAD

**Discussion by darefugee** [EdNekebno/DGTCentaurMods#164] [General]
Got the newer Centaur.  Board date is 9-19-2018.  Downloaded 1.2.1 installer and gave it a go.  First time, the display did not work but the website worked I think, I never looked at :5000 because I didn't realize it was on that port.  After looking at the webpages on http I saw nothing there that looked like a chess board, so I started again from scratch.  Wiped and reloaded the card with the installer.

Now it doesn't work.  Connected to monitor, I see firstboot.service fails every time, even when I start from a freshly configured microSD card.  I can ssh to the pi just fine, but the web site never starts.  The led circles just spin and spin.  Gave it 30 minutes to boot, still doesn't work.  

I tried wiping the card and reloading.  I tried deleting the files from my computer, downloading them from here again, and nothing works.  Tried 3 times yesterday, 4 times today.  Two different locations, three different wifi connections.  

Edit: looks like the pi is bad.  I'll get another and try again.

**Comment by wormstein** [comment]
Hi,
I am sorry to hear it's not working for you. Keeo in mind that the mod works only with the 32 bit version of Raspbian OS Lite.

There is nothing running on port 5000. The interface should load at http://ip_of_centaur_board/ Looks like you are accessing something else in your network.

One thing that apparently broke recently but we never had time to look into this is if you set a different user that "pi" setup will fail for some reason. Try stick with pi user for now when flashing the OS.

I don't know your experience with Linux but you should be able, as a last resort, to flash Raspbian OS, SSH connect to it and install the .deb file of the release. This won't copy over the original centaur software but at least will conirm that the hardware (soldering) it's all good.

Now I read your update about bad card. Anyway, see my suggestions too.

Feel free to join the discord server and we're happy to help you get it going.

All the best,

**Comment by darefugee** [comment]
Thanks, I'm on the discord server as well.

I did use the 32 bit version of Raspbian Lite, yes.  The one time I think
it worked, there was some rudimentary chess configuration options on the
website at http://(ip).  It was definitely not something else on my net.
It was chess but nothing that would allow me to start a game, or select
engines, or put the Centaur in board mode.  Nothing like that, just some
options to choose back-end configurations on the engines I think.  So I
thought maybe the web site install quit early and failed.  Then later I
looked at the docs more carefully and saw that the site was shown at
http://(ip):5000,
but by then it was too late and I'd already erased the card.  I'm pretty
sure my ham-handness broke the Pi, I see a tiny scratch on it that looks
deep enough to have gone through the traces.

Thanks for the tip about the default user, and also flashing the OS and
installing the .deb.  Might do that if the script continues to frustrate
me.  Going to get another Pi and try again.
